src/ui/profile_preferences.ui

```diff
@@ -1132,6 +1131,9 @@
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_foreground_color">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The foreground color used by default for the main console unless changed by a lua command or the MUD Server.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
How about just:

> The foreground color used for the main console by default

That will equally get the point across that it can be changed - and it doesn't really matter by what in this context, it's better to keep it shorter. Same for a few below

Reply:
If you want to keep the "unless" part, at least put it in brackets. Otherwise, this is a long sentence with no interpunction, which I'd like to avoid.

Reply:
@vadi2 elsewhere (like in the previously referred to colour trigger dialogue I do need to differentiate between triggers based on the **default** colour for both of the background/foreground set here and triggers based on **ignoring *one*** of them {it will be a warning message error to specify a colour trigger and then not specify at least *one* of them}).   

@Kebap - I get what you are saying and am minded to do that.

Reply:
In pending commit